packages/api-client/src/api/getVendor/index.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiContext, GetVendorParams, VendorSearchResult } from '../../types';
+import { deserializeVendor } from '../serializers/vendor';
+import axios from 'axios';
+
+export default async function getVendor({ config }: ApiContext, { vendorSlug }: GetVendorParams): Promise<VendorSearchResult> {
+  const response = await axios.get(
+    `${config.backendUrl}/api/v2/storefront/vendors/${vendorSlug}`
+  );
+  try {
+    const responseData = response.data;
+    const currentVendor = deserializeVendor(responseData.data);
+    return {
```

Comment:
Do we need this wrapper if we're searching for one specific vendor? We could as well return Promise<Vendor>